sdk/identity/azure-identity/src/samples/Azure Cache For Redis/Jedis/Azure-AAD-Authentication-With-Jedis.md

```diff
@@ -0,0 +1,207 @@
+## Azure Cache For Redis Azure AD With Jedis Client Library
+
+### Table of contents
+
+- [Jedis Library](#jedis-library)
+    - [Dependency Requirements](#dependency-requirements-jedis)
+    - [Authenticate with Azure AD - Hello World](#authenticate-with-azure-ad-jedis-hello-world)
+    - [Authenticate with Azure AD - Handle Re-Authentication](#authenticate-with-azure-ad-handle-re-authentication)
+    - [Authenticate with Azure AD - Azure Jedis Wrapper](#authenticate-with-azure-ad-azure-jedis-wrapper)
+
+### Jedis Library
+
+#### Dependency Requirements Jedis
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.5.0</version>
+</dependency>
+
+<dependency>
+    <groupId>redis.clients</groupId>
+    <artifactId>jedis</artifactId>
+    <version>4.0.1</version>
+</dependency>
+```
+
+#### Samples Guidance
+
+[Authenticate with Azure AD - Hello World](#authenticate-with-azure-ad-jedis-hello-world)
+This sample is recommended for users getting started to use Azure AD authentication with Azure Redis Cache.
+
+[Authenticate with Azure AD - Handle Re-Authentication](#authenticate-with-azure-ad-handle-re-authentication)
+This sample is recommended users looking to build long-running applications and would like to handle re authenticating with Azure AD upon token expiry.
+
+[Authenticate with Azure AD - Azure Jedis Wrapper](#authenticate-with-azure-ad-azure-jedis-wrapper)
+This sample is recommended for users looking to build long-running applications and would like to integrate our recommended wrapper implementation in their application which handles reconnection and re-authentication on user's behalf.
+
+
+#### Authenticate with Azure AD Jedis Hello World
+This sample is intended to assist in authenticating with Azure AD via Jedis client library. It focuses on displaying the logic required to fetch an Azure AD Access token and to use it as password when setting up the Jedis instance.
+
+Familiarity with the Jedis and Azure Identity client libraries is assumed. If you're new to the Azure Identity library for Java, see the docs for [Azure Identity](https://docs.microsoft.com/azure/developer/java/sdk/identity) and [Jedis](https://www.javadoc.io/doc/redis.clients/jedis/latest/index.html) rather than this guide.
```

Comment:
Instead of repeating this paragraph for each section, move it to the beginning of the doc.